README.md

```diff
@@ -1,148 +1,180 @@
+| Asetek       | Invicta              | 2433 | f300 |Not tested| ? | |
+| Asetek       | Forte                | 2433 | f301 |Not tested| ? | |
+| Asetek       | La Prima             | 2433 | f303 |Not tested| ? | |
+| Asetek       | Tony Kannan          | 2433 | f306 |Not tested| ? | |
+| Cammus       | C5                   | 3416 | 0301 | Silver   | hid-pidff[^1] | Works[^10] |
+| Fanatec      | CSL Elite            | 0eb7 | 0e03 | Gold     | hid-fanatecff[^7] | |
+| Fanatec      | CSL Elite (PS4)      | 0eb7 | 0005 | Gold     | hid-fanatecff[^7] | |
+| Fanatec      | DD                   | 0eb7 | 0020 | Gold     | hid-fanatecff[^7] | |
+| Fanatec      | DD Pro               | 0eb7 | 0020 | Gold     | hid-fanatecff[^7] | |
+| Fanatec      | Clubsport DD         | 0eb7 | 0020 | Gold     | hid-fanatecff[^7] | |
+| Fanatec      |ClubSport Wheel Base V2|0eb7 | 0001 | Silver   | hid-fanatecff[^7] | |
+| Fanatec      |ClubSport Wheel Base V2.5|0eb7|0004 | Silver   | hid-fanatecff[^7] | |
+| Fanatec      | Podium DD1           | 0eb7 | 0006 | Silver   | hid-fanatecff[^7] | |
+| Fanatec      | Podium DD2           | 0eb7 | 0007 | Silver   | hid-fanatecff[^7] | |
+| Fanatec      | CSR Elite            | 0eb7 | 0011 | Silver   | hid-fanatecff[^7] | |
+| Logitech     | MOMO Wheel           | 046d | c295 | Platinum | hid-logitech[^2] | Works |
+| Logitech     | DFP                  | 046d | c298 | Platinum | hid-logitech[^2] | Works |
+| Logitech     | DFGT                 | 046d | c29a | Platinum | hid-logitech[^2] | Works |
+| Logitech     | G25                  | 046d | c29b | Platinum | hid-logitech[^2] | Works |
+| Logitech     | G27                  | 046d | c29b | Platinum | hid-logitech[^2] | Works |
+| Logitech     | G29                  | 046d | c24f | Platinum | hid-logitech[^2] | Works |
+| Logitech     | G920                 | 046d | c262 | Silver   | hid-logitech-hidpp[^8] | Works |
+| Logitech     | G923 (Xbox edition)  | 046d | c26d | Silver   | hid-logitech-hidpp[^8] | Works |
+| Logitech     | G923 (PS edition)    | 046d | c267 | Platinum | hid-logitech[^3] | Works |
+| Logitech     | G Pro                | 046d | c272 | Bronze   | hid-logitech-hidpp[^4] | Works[^10] |
+| Moza         | R3                   | 346e | 0005 | Silver   | hid-pidff | Needs testing |
+| Moza         | R5                   | 346e | 0004 | Silver   | hid-pidff | Needs testing |
+| Moza         | R9                   | 346e | 0002 | Silver   | hid-pidff | Needs testing |
+| Moza         | R9v2                 | 346e | 0002 | Silver   | hid-pidff | Needs testing |
+| Moza         | R12                  | 346e | 0006 | Silver   | hid-pidff | Needs testing |
+| Moza         | R16                  | 346e | 0000 | Silver   | hid-pidff | Needs testing |
+| Moza         | R21                  | 346e | 0000 | Silver   | hid-pidff | Needs testing |
+| OpenFFBoard  |                    | 1209 | ffb0 | Silver | hid-pidff | Needs testing |
```

Comment:
I put silver to all wheels using hid-pidff because of the duration issue. It needs a patched driver or ffbwrap. I think there should be a readily available module with the fix to give it platinum, then we could link to that driver. I think it's in sync with the description for silver, it has a serious issue with a workaround.

Reply:
Aha, i understand now, ok :+1:

Reply:
The hid-pidff-fixed module which you mentioned in Matrix would make it much more easier to use these wheels. I think it's what would be needed to make them Platinum. I'm supposing anyone using the table will read the footnotes as advised so they're not only relying on the Silver rank and nothing more.